osu.Game/Screens/Menu/MenuTip.cs

```diff
@@ -94,14 +94,14 @@ private string getRandomTip()
         {
             string[] tips =
             {
-                "You can press Ctrl-T anywhere in the game to toggle the toolbar!",
-                "You can press Ctrl-O anywhere in the game to access options!",
+                "Press Ctrl-T anywhere in the game to toggle the toolbar!",
+                "Press Ctrl-O anywhere in the game to access options!",
                 "All settings are dynamic and take effect in real-time. Try changing the skin while watching autoplay!",
                 "New features are coming online every update. Make sure to stay up-to-date!",
                 "If you find the UI too large or small, try adjusting UI scale in settings!",
                 "Try adjusting the \"Screen Scaling\" mode to change your gameplay or UI area, even in fullscreen!",
                 "What used to be \"osu!direct\" is available to all users just like on the website. You can access it anywhere using Ctrl-B!",
-                "Seeking in replays is available by dragging on the difficulty bar at the bottom of the screen!",
+                "Seeking in replays is available by dragging on the difficulty bar at the bottom of the screen or by using the left and right arrow keys!",
```

Comment:
The term "progress bar" would be better for this hint:
```suggestion
                "Seeking in replays is available by dragging on the progress bar at the bottom of the screen or by using the left and right arrow keys!",
```